File: python/grading-program.py
student_scores = {
  "Harry": 81,
  "Ron": 78,
  "Hermione": 99, 
  "Draco": 74,
  "Neville": 62,
}

student_grades = {}

for student in student_scores:
    grade = ""
    value = student_scores[student]
    if value > 90:
        grade = "Outstanding"
    elif value > 80:
        grade = "Exceeds Expectations"
    elif value > 70:
        grade = "Acceptable"
    else:
        grade = "Fail"
    student_grades[student] = grade
    

print(student_grades)

File: go/grading-program.go
package main

import (
	"fmt"
)

func main() {
	student_scores := map[string]int{
		"Harry":    81,
		"Ron":      78,
		"Hermione": 99,
		"Draco":    74,
		"Neville":  62,
	}

	student_grades := map[string]string{}

	for key, value := range student_scores {
		var grade string
		if value > 90 {
			grade = "Outstanding"
		} else if value > 80 {
			grade = "Exceeds Expectations"
		} else if value > 70 {
			grade = "Acceptable"
		} else {
			grade = "Fail"
		}
		student_grades[key] = grade
	}

	fmt.Println(student_grades)
}
